sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -265,12 +263,19 @@ impl InputSelection {
             }
         }
 
+        let outputs = self
+            .added_outputs
+            .into_iter()
+            .chain(self.remainders.storage_deposit_returns)
+            .chain(self.remainders.data.iter().map(|r| r.output.clone()))
+            .collect::<Vec<_>>();
```

Comment:
Just checking, this doesn't include provided, is that correct?

Reply:
goof